specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3629 @@
+        }
+      },
+      "required": [
+        "resolutionKind",
+        "confidenceScore"
+      ]
+    },
+    "BaseExtraInformation":{
+      "type":"object",
+      "discriminator": "extraInformationKind",
+      "properties": {
+        "extraInformationKind":{
+          "type": "string",
+          "enum": [
+            "PersonNameExtraInfo",
+            "ColorExtraInfo",
+            "OrganizationExtraInfo"
+          ]
+        }
+      },
+      "required": [
+        "extraInformationKind"
+      ]
+    },
+    "PersonNameExtraInfo":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "properties":{
+        "firstName":{
+          "type":"string"
+        },
+        "middleName":{
+          "type":"string"
+        },
+        "lastName":{
+          "type":"string"
+        }
+      },
+      "description": "Breaking down the Person Name match into its subparts."
+    },
+    "ColorExtraInfo":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "properties":{
+        "normalizedValue":{
+          "type":"string"
+        }
+      },
+      "description": "A normalized value from the set of known colors."
+    },
+    "OrganizationExtraInfo":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "properties":{
+        "normalizedValue":{
+          "type":"string"
+        }
+      },
+      "description": "The normalized organization name in English"
+    },
+    "QuantityResolution":{
+      "description": "represents resolutions for quantities.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The numeric value that the extracted text denotes."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "TemporalSpanResolution": {
+      "allOf":[
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type":"object",
+      "properties": {
+        "begin":{
+          "$ref": "#/definitions/TimeExpression"
+        },
+        "end":{
+          "$ref": "#/definitions/TimeExpression"
+        },
+        "duration":{
+          "type":"string",
+          "description": "An optional duration value formatted based on the ISO 8601 (https://en.wikipedia.org/wiki/ISO_8601#Durations)"
+        },
+        "modifier":{
+          "$ref": "#/definitions/TemporalModifier"
+        },
+        "description": "represents the resolution of a date and/or time span."
+      }
+    },
+    "NumericRangeResolution": {
+      "allOf":[
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "rangeKind":{
```

Comment:
I think this isn't needed since the info should already be captured in the discriminator property `resolutionKind` defined in `BaseResolution`.